cloudy_mozdef/Makefile

```diff
@@ -27,10 +27,11 @@ all:
 	@echo 'Run ./dmake <target> in order to run the Makefile targets in Docker'
 
 # Note: This requires AWS access
+# https://blog.gruntwork.io/locating-aws-ami-owner-id-and-image-name-for-packer-builds-7616fe46b49a
 .PHONY: packer-build
 packer-build: ## Build the base AMI with packer
 	@echo "Branch based build triggered for $(BRANCH)..."
-	cd packer && packer -machine-readable build -var github_branch=${BRANCH} packer.json
+	ci/pack_and_copy ${BRANCH}
```

Comment:
Can you add a message that makes this reflect what we talked about yesterday?  It should notify the caller that this is being built using source "from github" and not local to set expectations.

Reply:
Good call, fixed in b85ed2a